### ISSUE TITLE
US 03.01.01

### ISSUE THREAD

**Issue by JulianCamiloGallego** [CMPUT301F22T04/Meal-Planner#16]
As a meal planner, I want to make a meal plan, for several upcoming days of meals, based on choosing from my recipes or individually from my ingredient storage.

**Comment by JulianCamiloGallego** [comment]
What is a meal plan supposed to look like?

**Comment by Dekr0** [comment]
Should a recipe and a ingredient (in ingredient storage) exist before adding it into a meal plan?

**Comment by JulianCamiloGallego** [comment]
Yeah I think it makes sense to only let the user pick from their recipes or ingredients. We could maybe some buttons to let the user create a recipe or ingredient from the meal plan page.
